docs/diagrams/BasicRecipeDiagram.puml

```diff
@@ -0,0 +1,125 @@
+@startuml BasicRecipeDiagram
+show members
+
+hide circle
+skinparam classAttributeIconSize 0
+
+class RecipeBook {
+    -recipes : UniqueRecipeList
+    +list()
+    +duplicate()
```

Comment:
what about methods to add / delete Recipes to the Recipe list?
+addRecipe()
+deleteRecipe()

Reply:
Fixed.